app/assets/javascripts/components/request.js.jsx

```diff
@@ -39,7 +39,10 @@ var Request = React.createClass({
         <Actions>
           <Action data={{ title: 'Resolve', action: this.props.resolve.bind(null, this.props.request.id)}} />
           <Action data={{ title: pinText, action: this.props.pin.bind(null, this.props.request.id)}} />
-          <Action data={{ title: 'Ping', action: this.props.bump.bind(null, this.props.request.id)}} />
+          <Ping 
+            data={{ title: 'Ping', action: (message) => this.props.bump(this.props.request.id, message)}}
+            id={moment.utc(this.props.request.created_at).valueOf()}
```

Comment:
Why not use `this.props.request.id` for a unique ID here?

Reply:
Oops, forgot about that somehow. Fixed!